packages/@react-spectrum/icon/docs/Icon.mdx

```diff
@@ -25,12 +25,32 @@ category: Misc
 ---
 
 # Icon
-<p>Docs coming soon</p>
+
+<p>{docs.exports.Icon.description}</p>
 
 ## Available Icons
 
 ### Workflow
-<HeaderTable name={workflowIconPackageData.name} version={workflowIconPackageData.version} />
+
+<HeaderInfo
```

Comment:
This should go at the top of the page. I think rather than having two sections for workflow and color icons, we'd have two pages instead.

Reply:
sounds good